_posts/2019-11-04-css-is-hard-and-vanguard-2019-learnings.md

```diff
@@ -0,0 +1,211 @@
+updated on the progress of our work. Despite our best intentions, it’s difficult to work on two teams at once and
+feel fully committed to either.
+
+## **2. Estimation is Hard**
+
+Due to the way this project was organized, we allotted 1.5 sprint cycles to complete the engineering work. However,
+this time allotment was decided on before we had received the completed design specifications from the design firm.
+This posed a set of interesting challenges in the sense that the Grow team committed to delivering a feature by a
+predetermined deadline without having all of the information needed to understand the full scope of the project. As
+with any engineering project, there are often unknown unknowns, those pesky little gotchas that surface as work
+commences on a project, but that cannot always be predicted. This presented an interesting lesson in learning how
+to estimate the amount of work required to deliver a minimum viable product and reduce the scope to meet certain
+deadlines.
+
+Given the limited amount of time to deliver some fairly complex designs, it was clear that we would have to reduce
+some of the technical complexity in order to meet the deadline. This is pretty standard in agile development, but
+this was the first time that I was in a position of having to deliver this information. . I felt fully supported by
+my managers, tech lead, and colleagues in having to deliver the news that we were not going to be able to deliver a
+feature identical to the one given to us by the design firm. From this experience, I learned that making accurate
+estimations about how much work a project actually involves is hard, relying on agile as a rule makes it easier,
+communicating early and often when there’s any uncertainty around delivering a feature by deadline is critical, and
+working with team members that value candor, flexibility, communication, and iteration is empowering.
+
+## **3. CSS is Hard**
+
+I get excited about implementing visually delightful designs, and Artsy Vanguard’s 2019 designs did not disappoint.
+There were two key components that involved interesting CSS approaches:
+
+### - Mixed Blending Mode to handle the inversion of the side frame text when it appeared over different backgrounds.
+
+### - Creating the visual effect of `.mp4` videos nested inside of SVG shapes
+
+**_Mixed Blend Mode_**
+
+Mixed blend mode is a CSS property that handles blending the colors of overlapping elements. In this case, we
+wanted to create the effect of inverting the colors of the static side frame text when they overlapped with either
+the white background, the black SVG shapes, or the SVG videos.
+
+This inverted color effect also needed to be applied to the text of the editorial content when it overlapped with
+the black SVG background, when it extended past the SVG background onto a white background, and when the current
+article was in an expanded state, such as when a user selected “Read More”.
```

Comment:
I think the last part of this sentence could be simplified to reduce length. Something like

"...onto a white background, and when the article was displaying its entire body."

Maybe not exactly that, but I feel like the detail of "such as when a user selected 'read more'" is an implementation detail that readers might get distracted by.